test/fixtures/controller.line/borderColor/value.js

```diff
@@ -20,13 +20,16 @@ module.exports = {
 			title: false,
 			elements: {
 				line: {
+					borderColor: '#00ff00',
 					fill: false,
 				},
 				point: {
-					borderColor: '#00ff00',
```

Comment:
I would keep that line and use `#0000ff` so we can see the fallback for the second dataset.